### DISCUSSION TITLE
2023-09-22

### DISCUSSION THREAD

**Discussion by yeopto** [woowacourse-teams/2023-haru-study#562] [daily_meeting]
### 오늘의 할 일
- 5차 데모 발표

**Comment by yeopto** [comment]
### 체크인

10점 - 숙면을 취해서 너무 좋습니다

**Comment by aak2075** [comment]
### 체크인

10점 - 불금입니다

**Comment by jaehee329** [comment]
### 체크인
10점 - 많이 못 잤지만 금요일이잖아~

**Comment by woosung1223** [comment]
### 체크인
8점 - 피곤하지만 금요일이라서 8점 입니닫

**Comment by nlom0218** [comment]
### 체크인

9점 - 테헤란 누나가 꽁짜 토스트 주었지만 제 입맛은 아니었어요. 건강한 맛이었습니다.

**Comment by woo-jk** [comment]
### 체크인 
2점 - 발표 대본을 못썼어요

**Reply by jaehee329** [reply]
화이팅입니다

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 이젠 아무 생각이 안드네요
